apps/video/js/video.js

```diff
@@ -136,6 +136,22 @@ document.addEventListener('visibilitychange', function visibilityChange() {
   }
 });
 
+// Handle interruption from audio channel and set correct
+// play/pause button. See Bug 121457
+dom.player.addEventListener('mozinterruptbegin', function() {
+  if (playing) {
+    setButtonPaused(true);
+    playing = false;
+  }
+});
+
+dom.player.addEventListener('mozinterruptend', function() {
+  if (!playing) {
```

Comment:
Nit: it seems we get 'mozinterruptend' only when a video is not playing so this check appears to be redundant. With that said, it does make the code more readable.